src/scss/components/_dialog.scss

```diff
@@ -15,20 +21,21 @@
     flex: 1 0 auto;
     padding: 30px;
     position: relative;
-    background-color: get-color(air);
+    background-color: get-color(breeze);
     border-radius: 10px;
     align-items: center;
     justify-content: center;
     font-style: italic;
+    color: get-color(rastaGray);
 
     &::after {
```

Comment:
@MarcoBrunoBR I would if we had the same usage in other components. Now only dialogs uses these configs.